account_invoice_validate_send_email/views/res_company_views.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="view_company_form" model="ir.ui.view">
+        <field name="name">res.company.form</field>
+        <field name="model">res.company</field>
+        <field name="inherit_id" ref="base.view_company_form" />
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='website']" position="after">
```

Comment:
@tack-yabeQuartile I suppose the field presentation is a bit broken if we insert the field here.  Can you find a better place to put it in?